app/models/form/lettings/questions/age.rb

```diff
@@ -1,9 +1,8 @@
 class Form::Lettings::Questions::Age < ::Form::Question
-  def initialize(id, hsh, page, person_index:, person_type:)
```

Comment:
Removed person_type and all the duplicate pages, because it was only being used to determine pre 2024 hint. Individual questions for 2023 should no longer be accessible, so this doesn't really matter + removing this makes copy file simpler